examples/react-spa/src/Settings.tsx

```diff
@@ -14,23 +14,21 @@ export const Settings = () => {
 
   const navigate = useNavigate()
 
-  const createFlow = useCallback(
-    () =>
-      sdk
-        // create a new settings flow
-        // the flow contains the form fields, error messages and csrf token
-        // depending on the Ory Network project settings, the form fields returned may vary
-        .createBrowserSettingsFlow()
-        .then(({ data: flow }) => {
-          setFlow(flow)
-        })
-        // something serious went wrong so we redirect to the settings page
-        .catch((err) => {
-          console.error(err)
-          navigate("/settings", { replace: true })
-        }),
-    [],
-  )
+  const createFlow = () => {
+    sdk
+      // create a new settings flow
+      // the flow contains the form fields, error messages and csrf token
+      // depending on the Ory Network project settings, the form fields returned may vary
+      .createBrowserSettingsFlow()
+      .then(({ data: flow }) => {
+        setFlow(flow)
+      })
+      // something serious went wrong, so we redirect to the settings page
+      .catch((err) => {
+        console.error(err)
+        navigate("/settings", { replace: true })
+      })
```

Comment:
We will need to handle the errors here properly as well. 

In case of a no session error we will get an infinite redirect loop.
https://github.com/ory/kratos/blob/master/selfservice/flow/settings/handler.go#L283-L288

Reply:
After your first comment about the 410 etc, I started to check all the error codes and how to handle them. I have created a generic error handler that will handle 410 with `use_flow_id` or navigate to `/`.

Probably will have a commit later today of that error handling and a few other things. After that needs a closer look if everything is handled :)